dev-libs/highway/highway-1.2.0.recipe

```diff
@@ -13,23 +13,33 @@ Why Highway?
 HOMEPAGE="https://github.com/google/highway"
 COPYRIGHT="2020 Google LLC"
 LICENSE="Apache v2"
-REVISION="2"
-SOURCE_URI="https://github.com/google/highway/archive/refs/tags/$portVersion.tar.gz"
-CHECKSUM_SHA256="e1d47ce510429fdcf31f41697ca74fb0dcd59d933196e531a86d51751a56f4cc"
+REVISION="1"
+SOURCE_URI="https://github.com/google/highway/releases/download/$portVersion/highway-$portVersion.tar.gz"
+CHECKSUM_SHA256="58e9d5d41d6573ad15245ad76aec53a69499ca7480c092d899c4424812ed906f"
 SOURCE_FILENAME="highway-$portVersion.tar.gz"
 
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
-	highway$secondaryArchSuffix = $portVersion
-	devel:libhwy$secondaryArchSuffix = $portVersion
-	devel:libhwy_contrib$secondaryArchSuffix = $portVersion
+	highway$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libhwy$secondaryArchSuffix = $portVersion
+	lib:libhwy_contrib$secondaryArchSuffix = $portVersion
+	lib:libhwy_test$secondaryArchSuffix = $portVersion
```

Comment:
Use libVersionCompat here (libVersion = 1.2.0 compat = 1) and for all the libraries in this package (including _devel)